src/components/profile/style/profile.less

```diff
@@ -1,3 +1,6 @@
+/*library added until we integrate the icons*/
+@import url("https://netdna.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css");
```

Comment:
The icons are integrated in master. So you can remove this reference and rebase onto current master. @mariavkourouni